src/test/java/uk/gov/companieshouse/service/DissolutionPatcherTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.companieshouse.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.fixtures.DissolutionFixtures;
+import uk.gov.companieshouse.mapper.DissolutionRequestMapper;
+import uk.gov.companieshouse.mapper.DissolutionResponseMapper;
+import uk.gov.companieshouse.model.db.Dissolution;
+import uk.gov.companieshouse.model.dto.DissolutionCreateRequest;
+import uk.gov.companieshouse.model.dto.DissolutionCreateResponse;
+import uk.gov.companieshouse.model.dto.DissolutionPatchResponse;
+import uk.gov.companieshouse.repository.DissolutionRepository;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class DissolutionPatcherTest {
+
+    @InjectMocks
+    private DissolutionPatcher patcher;
+
+    @Mock
+    private DissolutionRepository repository;
+
+    @Mock
+    private DissolutionResponseMapper responseMapper;
+
+    @Test
+    public void patch_addsApprovalToDirector_savesInDatabase_returnsPatchResponse() {
```

Comment:
This test could be beefed out. We are missing:
* A verify call to ensure that the data was in fact saved in the database
* Some assertions to ensure that the approval was added to the dissolution before it was saved into the database